.github/workflows/build-package.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Build release package
         run: |
           ./build-source.sh
-          ./build.sh build package release
+          ./build.sh CONFIGURE_OPTS="-Dwerror=true" build package release
```

Comment:
Just `...build -Dwerror=true ... ` then.

Reply:
Keep in mind that the `-Dwerror` option is meson, so it need to be propagated within the `build.sh` file. Are you OK with that or have something else in mind?

Reply:
`build` "command" passes dash-prefixed stuff to meson itself. Well, it _should_ anyway.

Reply:
Indeed it does pass them along, thanks.